From 40942d76840add267793234a3124fe1301feae55 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <xnox@ubuntu.com>
Date: Thu, 10 Dec 2020 13:42:13 +0000
Subject: UBUNTU: fixup ofnet initialization.

Signed-off-by: Dimitri John Ledkov <xnox@ubuntu.com>

Patch-Name: ubuntu-fixup-rhboot-f34-support-non-ethernet.patch
(cherry picked from commit a819547bd60ba1c6088977b7bfe288f4f867fad9)
---
 grub-core/net/drivers/ieee1275/ofnet.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/grub-core/net/drivers/ieee1275/ofnet.c b/grub-core/net/drivers/ieee1275/ofnet.c
index 057ba8597..4621b0a34 100644
--- a/grub-core/net/drivers/ieee1275/ofnet.c
+++ b/grub-core/net/drivers/ieee1275/ofnet.c
@@ -154,7 +154,7 @@ grub_ieee1275_parse_bootpath (const char *devpath, char *bootpath,
   char *equal_char = 0;
   grub_size_t field_counter = 0;
   grub_net_network_level_address_t client_addr = {0, {0}, 0}, gateway_addr = {0, {0}, 0}, subnet_mask = {0, {0}, 0};
-  grub_net_link_level_address_t hw_addr = {0, {{0, 0, 0, 0, 0, 0}}};
+  grub_net_link_level_address_t hw_addr = {0, 0, {{0, 0, 0, 0, 0, 0}}};
   grub_net_interface_flags_t flags = 0;
   struct grub_net_network_level_interface *inter = NULL;
   grub_uint16_t vlantag = 0;
